cargo-near/src/build.rs

```diff
@@ -68,7 +78,7 @@ pub(crate) fn run(args: BuildCommand) -> anyhow::Result<()> {
     wasm_artifact.path = util::copy(&wasm_artifact.path, &out_dir)?;
 
     // todo! if we embedded, check that the binary exports the __contract_abi symbol
-    println!("{}", "Contract Successfully Built!".green().bold());
+    step_printer.print_step(&format!("{}", "Contract Successfully Built!"));
```

Comment:
... so this would either revert to:

```suggestion
    step_printer.print_step(&"Contract Successfully Built!".green().bold());
```

Otherwise it can be simplified into:

```suggestion
    step_printer.print_step("Contract Successfully Built!");
```